code-challenges.js

```diff
@@ -16,41 +22,137 @@ const fruit2 = "banana"
 const fruit3 = "cherry"
 const fruit4 = "kiwi"
 
+const stringCombo = (string1, string2) => {
+    if(string1.length > string2.length) {
+        return string1
+    } else if(string1.length < string2.length) {
+        return string2
+    } else {
+        return "Oops, neither are greater"
+    }
+}
+
+console.log(stringCombo(fruit1, fruit2)) // returned banana
+console.log(stringCombo(fruit3, fruit4)) // returned cherry
+
 
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-// Pseudo code:
+// Pseudo code: 
+// Declare a function called boilingPoint
+// Set parameter of temp to pass through the function for evaluation
+// Write conditional statements to compare parameter to argument and return a value
+    //These statements will compare the parameter of temperature to the arguement or arguments it is passed through until it meets the correct criteria and returns the appropriate response.
+    // I will start with an if statement, then else if, then move to an else statement.
+    // Use string interpolation to recall the const for the return the const value plus a string
+// After the function is working properly, I will invoke it and pass the 3 const variables as parameters and log the returned value/ response.
+//
 
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+const boilingPoint = (temp) => {
+    if(temp < 212) {
+        return `${temp1}` + " is below boiling point"
+    } else if(temp === 212) {
+        return `${temp2}` + " at boiling point"
+    } else(temp > 212) 
+        return `${temp3}` + " above boiling point"
+ } 
+
+console.log(boilingPoint(temp1)) // returned "42 below boiling point"
+console.log(boilingPoint(temp2))// returned "350 above boiling point"
+console.log(boilingPoint(temp3))// returned "212 at boiling point"
```

Comment:
This looks great and your pseudocode is excellent. My recommendation to improve the code a little bit would be to use string interpolation for the entire sentence.  Currently it looks like you are using a combination of interpolation and concatenation.